lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -217,35 +217,31 @@ int autolink_extract_main(ArrayRef<const char *> Args, const char *Argv0,
   // in most object files
 
   std::vector<std::string> SwiftRuntimeLibsOrdered = {
-      // Common Swift runtime libs
-      "-lswiftSwiftOnoneSupport",
-      "-lswiftCore",
-      "-lswift_Concurrency",
+      "-lFoundationXML",
+      "-lFoundationNetworking",
+      "-lFoundation",
+      "-lswiftDispatch",
+      "-lswiftRegexBuilder",
+      "-lswift_Backtracing",
       "-lswift_StringProcessing",
-      "-lswift_RegexBuilder",
       "-lswift_RegexParser",
-      "-lswift_Backtracing",
+      "-lswift_Concurrency",
       "-lswiftGlibc",
-      "-lBlocksRuntime",
-      // Dispatch-specific Swift runtime libs
+      "-lswiftSwiftOnoneSupport",
+      "-lswiftCore",
+      "-lCoreFoundation",
+      "-licui18nswift",
+      "-licuucswift",
+      "-licudataswift",
       "-ldispatch",
+      "-lBlocksRuntime",
       "-lDispatchStubs",
-      "-lswiftDispatch",
-      // CoreFoundation and Foundation Swift runtime libs
-      "-lCoreFoundation",
-      "-lFoundation",
-      "-lFoundationNetworking",
-      "-lFoundationXML",
-      // Foundation support libs
       "-lcurl",
       "-lxml2",
       "-luuid",
       // XCTest runtime libs (must be first due to http://github.com/apple/swift-corelibs-xctest/issues/432)
```

Comment:
Might as well also fix my silly comment typo from way back in my original PR while we're at it:
```suggestion
      // XCTest runtime libs (must come after runtime libs due to https://github.com/apple/swift-corelibs-xctest/issues/432)
```